examples/node-app-mssql/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It looks like the example wasn't done in TS, it was plain JS but now we've mistakenly assumed and created TS based migrations - I'm not sure how it's working since the `package.json` doesn't have typescript on it :smile:.

I'd say let's keep everything plain JS and it would be simple.